src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java

```diff
@@ -152,7 +152,6 @@ public void summaryReport_HappyPath() throws Exception {
                             .andExpect(model().attribute("sectionId", is(sectionId.toString())))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "sectionId" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)